clients/src/main/java/org/apache/kafka/clients/consumer/internals/ConsumerCoordinator.java

```diff
@@ -648,10 +648,14 @@ private void maybeUpdateGroupSubscription(String assignorName,
         updateGroupSubscription(allSubscribedTopics);
 
         isLeader = true;
-        assignmentSnapshot = metadataSnapshot;
 
-        if (skipAssignment)
+
+        if (skipAssignment) {
+            log.info("Skipped assignment for returning static leader at generation {}. The static leader " +
+                "will collect its existing assignment.", generation().generationId);
```

Comment:
I'm not sure if we need to put the 2nd sentence. Maybe the 1st one is enough? Or maybe change the 2nd one with:
`The static leader will collect its existing assignment with empty assignment syncGroup request.` 

WDYT?

Reply:
I would not mention the sync group request. How about `The static leader will continue with its existing assignment`?